src/main/resources/rs117/hd/scene/model_overrides.json

```diff
@@ -16322,24 +16329,333 @@
   {
-    "description": "Hunter Guild cave walls",
+    "description": "Perilous Moons Murals",
+    "baseMaterial": "GRUNGE_2",
+    "objectIds": [
+      51380,
+      51382,
+      51384
+    ]
+  },
+  {
+    "description": "Perilous Moons Statues",
+    "baseMaterial": "MARBLE_2_GLOSS",
+    "objectIds": [
+      51309,
+      51310,
+      51311,
+      51329,
+      51330,
+      51332,
+      51333,
+      51334,
+      51336,
+      51337,
+      51338,
+      51339,
+      51340,
+      51341,
+      51372,
+      51373,
+      51374
+    ],
+    "uvType": "BOX",
+    "uvScale": 0.5
+  },
+  {
+    "description": "Perilous Moons Giant Eagle Statue, separate because for no gloss",
+    "baseMaterial": "MARBLE_2",
+    "objectIds": [
+      51331
+    ],
+    "uvType": "BOX",
+    "uvScale": 0.7
+  },
+  {
+    "description": "Perilous Moons Braziers and Chains",
+    "baseMaterial": "GRUNGE_2",
+    "objectIds": [
+      25121,
+      51342,
+      51343,
+      51344,
+      51345,
+      51345,
+      51398,
+      51399
+    ]
+  },
+  {
+    "description": "Perilous Moons Plants",
+    "baseMaterial": "PLANT_GRUNGE_1",
+    "objectIds": [
+      36220,
+      39681,
+      39682,
+      51081,
+      51082,
+      51083,
+      51084,
+      51257,
+      51258,
+      51261,
+      51262,
+      51263,
+      51264,
+      51326,
+      51327,
+      51328,
+      51358,
+      51365,
+      51433,
+      51434,
+      51435,
+      51436,
+      51437,
+      51438,
+      51466,
+      51467,
+      51468,
+      51470,
+      51471,
+      51472,
+      51473,
+      51585,
+      51586
+    ]
+  },
+  {
+    "description": "Streambound Cavern Bridges",
+    "baseMaterial": "WOOD_GRAIN_2",
+    "objectIds": [
+      51348,
+      51349,
+      51350,
+      51351,
+      51352,
+      51353,
+      51354,
+      51355,
+      51356,
+      51357
+    ],
+    "uvType": "MODEL_XZ",
+    "uvScale": 0.9
+  },
+  {
+    "description": "Perilous Moons Monolith and Gates",
+    "baseMaterial": "METALLIC_1_LIGHT_HIGHGLOSS",
+    "objectIds": [
+      51379,
+      51392,
+      51393
+    ]
+  },
+  {
+    "description": "Perilous Moons Carvings",
     "baseMaterial": "ROCK_3",
     "objectIds": [
+      51400
+    ],
+    "uvType": "BOX",
+    "uvScale": 0.5
+  },
+  {
+    "description": "Perilous Moons Ancient Shrine Dirt",
+    "baseMaterial": "DIRT_2",
+    "objectIds": [
+      51097,
+      51098,
+      51272,
+      51273,
+      51274,
+      51275,
+      51276,
+      51277,
+      51279,
+      51280,
+      51281,
+      51282,
+      51283,
+      51284,
+      51285,
+      51286,
+      51287,
+      51288,
+      51289,
+      51290,
+      51291,
+      51292,
+      51293,
+      51294,
+      51295,
+      51296,
+      51297,
+      51298,
+      51299
+    ],
+    "uvType": "MODEL_XZ",
+    "uvScale": 0.75
+  },
+  {
+    "description": "Perilous Moons Ancient Shrine stone chips, Hide them they are ugly here",
```

Comment:
This presumably needs to be area-specific, if you want to hide these only in some cases

Reply:
Also, forgot to mention, it's better to use `"hide": true` than a transparent material.

Reply:
I can resolve these tonight

Reply:
This should be fixed.